ruleset/sca/amazon/cis_amazon_linux_2022.yml

```diff
@@ -0,0 +1,5084 @@
+    compliance:
+      - cis: ["1.1.10"]
+      - cis_csc_v8: ["10.3"]
+      - cis_csc_v7: ["13.7"]
+      - nist_sp_800-53: ["SC-18(4)"]
+      - cmmc_v2.0: ["MP.L2-3.8.7"]
+      - hipaa: ["164.310(d)(1)"]
+    condition: all
+    rules:
+      - "c:modprobe -n -v usb-storage -> r:install /bin/true"
+      - "not c:lsmod -> r:usb-storage"
+
+  ############################################################
+  # 1.2 Configure Software Updates.
+  ############################################################
+
+  # 1.2.1 Ensure GPG keys are configured -> can't be implemented - requires manual action.
+
+  # 1.2.2 Ensure gpgcheck is globally activated
+  - id: 31033
+    title: "Ensure gpgcheck is globally activated."
+    description: "The gpgcheck option, found in the main section of the /etc/dnf/dnf.conf and individual /etc/yum/repos.d/* files determines if an RPM package's signature is checked prior to its installation."
+    rationale: "It is important to ensure that an RPM's package signature is always checked prior to installation to ensure that the software is obtained from a trusted source."
+    remediation: "Edit /etc/dnf/dnf.conf and set ' gpgcheck=1 ' in the [main] section. Edit any failing files in /etc/yum.repos.d/* and set all instances of gpgcheck to ' 1 '."
+    compliance:
+      - cis: ["1.2.2"]
+      - cis_csc_v8: ["7.3"]
+      - cis_csc_v7: ["3.4"]
+      - nist_sp_800-53: ["SI-2"]
+      - cmmc_v2.0: ["SI.L1-3.14.1"]
+      - pci_dss_3.2.1: ["6.2"]
+      - soc_2: ["CC7.1"]
+    condition: all
+    rules:
+      - "f:/etc/dnf/dnf.conf -> r:gpgcheck=1"
+      - "not c:grep -Rh ^gpgcheck /etc/yum.repos.d/ -> r:gpgcheck=0"
+
+  # 1.2.3 Ensure package manager repositories are configured -> can't be implemented - requires manual action.
+
+  ############################################################
+  # 1.3 Filesystem Integrity Checking.
+  ############################################################
+
+  # 1.3.1 Ensure AIDE is installed.
+  - id: 31034
+    title: "Ensure AIDE is installed."
+    description: "AIDE takes a snapshot of filesystem state including modification times, permissions, and file hashes which can then be used to compare against the current state of the filesystem to detect modifications to the system."
+    rationale: "By monitoring the filesystem state compromised files can be detected to prevent or limit the exposure of accidental or malicious misconfigurations or modified binaries."
+    remediation: "Run the following command to install AIDE: # dnf install aide . Configure AIDE as appropriate for your environment. Consult the AIDE documentation for options. Initialize AIDE: # aide --init && mv /var/lib/aide/aide.db.new.gz /var/lib/aide/aide.db.gz"
```

Comment:
There are extra spaces before the end of the sentence dots.

Reply:
Solved